tests/Aspire.Microsoft.Azure.Cosmos.Tests/Aspire.Microsoft.Azure.Cosmos.Tests.csproj

```diff
@@ -9,6 +9,7 @@
 
     <ProjectReference Include="..\..\src\Components\Aspire.Microsoft.Azure.Cosmos\Aspire.Microsoft.Azure.Cosmos.csproj" />
     <ProjectReference Include="..\Aspire.Components.Common.Tests\Aspire.Components.Common.Tests.csproj" />
+    
```

Comment:
Changes like this seem to have slipped in. I think removing them should help reduce the size of the diff too.

Reply:
This looks more like a rendering bug - I only removed excessive spacing
![image](https://github.com/user-attachments/assets/1393c3e6-52d5-42cc-996b-a135a07eb33b)

I'll fix these, however, I agree these look untidy.